docfx_yaml/extension.py

```diff
@@ -455,11 +533,52 @@ def insert_children_on_class(app, _type, datam):
             insert_class.append(datam)
 
 
+def insert_children_on_function(app, _type, datam):
+    """
+    Insert children of a specific class
+    """
+    if FUNCTION not in datam:
+        return
+
+    insert_functions = app.env.docfx_yaml_functions[datam[FUNCTION]]
+    insert_functions.append(datam)
+
+
+def insert_page(app, _type, datam):
+    """
+    Insert a page in docfx_yaml_pages.
+    """
+    if PAGE not in datam:
+        return
+
+    insert_pages = app.env.docfx_yaml_pages[datam[PAGE]]
+    insert_pages.append(datam)
+
+
+def _process_cross_references_in_conceptual_pages(app):
```

Comment:
Don't think this will work. Cuz after onboarding the package, Conceptual contents may not be put together with the API reference. They might event be put into different docsets, thus we should ref to conceptual pages using http link.